.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+- type: textarea
+  id: verification_steps
+  attributes:
+    label: Verification Steps
+    description: How the fix can be verified once the cherry-pick has been performed.
+    placeholder: Provide detailed steps to manually verify the changes in this cherry-pick. They will be run by a developer or a QA expert to ensure that the bug was indeed fixed.
+  validations:
+    required: true
+- type: textarea
+  id: cherry_pick_progress
+  attributes:
+    label: Cherry-Pick Progress
+    description: Progress details for this cherry-pick.
+    value: |
+      <!-- Leave this section as it is for now. -->
+
+      To be updated by @ampproject/release-on-duty as each stage is completed.
+      - [x] Cherry-pick request created
+      - [ ] Cherry-pick request approved
+      - [ ] New version released to Beta / Experimental channels
+      - [ ] New version released to Stable channel
+      - [ ] New version released to LTS channel
+      - [ ] Release tracker updated
+      - [ ] Cherry-pick completed
+- type: markdown
+  id: mini_postmortem
+  attributes:
+    value: |
+      ## Mini Postmortem
+      The following sections serve as a postmortem for this cherry-pick, and are meant to document things like the bug summary, its root causes, user impact, and action items. Fill in these sections to the best of your knowledge so that similar issues can be prevented in future.
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary
+    description: A summary of the problem and its root cause(s).
+    placeholder: |
+      Provide answers to questions like:
+      - What is the bug being fixed?
+      - What are its root cause(s)?
+      - How did it end up getting released to stable channel?
+      If details are as yet unknown, put down "TODO" in this section and remember to fill it in later.
+  validations:
+    required: true
+- type: textarea
+  id: impact
+  attributes:
+    label: Impact
+    description: How users were impacted.
+    placeholder: |
+      Provide answers to questions like:
+      - Which users were affected? E.g. Users of Firefox version XXX.
+      - Roughly how many users? E.g. O(YYY) users.
+      - How were they affected? E.g. ZZZ feature did not work.
+      If details are as yet unknown, put down "TODO" in this section and remember to fill it in later.
+  validations:
+    required: true
+- type: textarea
+  id: action_items
+  attributes:
+    label: Action Items
+    description: What can be done to prevent this in future.
+    placeholder: |
+      Provide answers to questions like:
+      - How can this class of bugs be prevented in the future?
+      - How do we detect them sooner and mitigate their impact?
+      - How could we make the investigation of these issues easier?
+      If details are as yet unknown, put down "TODO" in this section and remember to fill it in later.
+  validations:
+    required: true
+- type: textarea
+  id: notifications
```

Comment:
Should a requester be able to change `/cc @ampproject/release-on-duty @ampproject/wg-approvers @ampproject/cherry-pick-approvers`? Suggestion to keep that part static while allowing additional cc's if desired... or keep this form short by removing this.

Reply:
The only available static type is `markdown`, which [gets dropped](https://gh-community.github.io/issue-template-feedback/structured/#markdown) from the issue body when the form is submitted. Looks like the only way to include pre-filled text is via the `value` attribute.

I think this is okay because it's no worse than what we have today, and there is an incentive to keep the notification list intact for speedy approval / execution.